### PR TITLE
Notification sound

### PR DESCRIPTION
**Pull request by brooke-ec**
- Updated URLs to not explicitly define the host, this falls back to the current host. This makes it easier to test in a local development environment.
- Added a notification sound to be played when the page is not focused (hopefully sounds like a cat!)
- Added a favicon which changes to signal unread messages.

### CODE REVIEW

File: Assets/static/HTML/Application.html [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
 
-    <link rel="stylesheet" href="http://13.40.94.202/styles?path=common.css">
-    <script src="http://13.40.94.202/script?path=application.js"></script>
-    <script src="http://13.40.94.202/script?path=login.js"></script>
-    <script src="http://13.40.94.202/script?path=messenger.js"></script>
-
-</head>
-<body>
+    <link rel="stylesheet" href="/styles?path=common.css" />
+    <script src="/script?path=application.js"></script>
```
Comment (Nathcat) [inline Assets/static/HTML/Application.html:10]: Lowkey forgot this was a thing 😂😂

File: Assets/static/scripts/messenger.js [hunk 2]
```diff
@@ -52,6 +52,11 @@ function message_to_html(message, displayName) {
 function push_message(messages, i) {
     if (i == messages.length) return;
 
+    if (!VISIBLE) {
+        document.getElementById("icon").href = "/images?path=favicon_notification.png";
+        new Audio("http://127.0.0.1:8080/sounds?path=notification.mp3").play();
```
Comment (Nathcat) [inline Assets/static/scripts/messenger.js]: Can the URL here be `/sounds?path=notification.mp3`? Or does it have to include the hostname as well?

**Review by Nathcat — COMMENTED** [review]
Just one comment about the URL for requesting the audio, otherwise seems good to me.

**Review by Nathcat — APPROVED** [review]

File: Assets/static/scripts/messenger.js (hunk 2 above)
Comment (brooke-ec) [reply Assets/static/scripts/messenger.js]: Ah, my mistake! No, the hostname is not required there! Thats what I get for making a pull request at 3am 😅

Comment (Nathcat) [reply Assets/static/scripts/messenger.js]: Haha no worries lol. I forgot that you could even omit the host name so 😂😂

**Pull request merged**